src/client/components/app/style.css

```diff
@@ -1,5 +1,9 @@
 @import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,500');
 
+* {
+	box-sizing: border-box;
```

Comment:
Another options is this, which allows for easily resetting the `box-sizing`:

```css
html {
  box-sizing: border-box;
}

*, *:before, *:after {
  box-sizing: inherit;
}
```

Reference: https://css-tricks.com/box-sizing/#article-header-id-6